google-cloud-spanner/src/test/java/com/google/cloud/spanner/connection/StatementTimeoutTest.java

```diff
@@ -1048,9 +1110,9 @@ public void run() {
               },
               WAIT_BEFORE_CANCEL,
               TimeUnit.MILLISECONDS);
-
-      expected.expect(SpannerExceptionMatcher.matchCode(ErrorCode.CANCELLED));
       connection.runBatch();
```

Comment:
Add a `fail("Expected exception")` to prevent the test case from succeeding in case of no error.

Reply:
Done.